pulsar/consumer_partition.go

```diff
@@ -1594,6 +1607,13 @@ func (pc *partitionConsumer) runEventsLoop() {
 				pc.log.Info("close consumer, exit reconnect")
 				return
 			case connectionClosed := <-pc.connectClosedCh:
+				if pc.seeking.Load() {
+					go func() {
+						<-time.After(100 * time.Millisecond)
+						pc.connectClosedCh <- connectionClosed
```

Comment:
Does the Java consumer have the same issue?

Reply:
Java client has been fixed this issue

Reply:
Ok. I see that the Java client will wait a while before reconnecting to the broker:
https://github.com/apache/pulsar/blob/83b86abcb74595d7e8aa31b238a7dbb19a04dde2/pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConnectionHandler.java#L200

And it's the same for the go client: 
https://github.com/apache/pulsar-client-go/blob/682bf5fde149754c1648fbffccd39c2c51f17551/pulsar/consumer_partition.go#L1719

So it doesn't seem the go client should have this issue, right?

Reply:
Fixed by this PR.